Formula/daemonize.rb

```diff
@@ -18,4 +18,8 @@ def install
     system "make"
     system "make", "install"
   end
+
+  test do
+    system "#{bin}/daemonize"
```

Comment:
Can we have this test do something real? Or at minimum assert match the version?